modules/storages/app/components/storages/admin/edit_form_header_component.html.erb

```diff
@@ -43,5 +43,20 @@ See COPYRIGHT and LICENSE files for more details.
       button.with_leading_visual_icon(icon: :trash)
       I18n.t("button_delete")
     end
+    header.with_tab_nav(label: nil, test_selector: :project_attribute_detail_header) do |tab_nav|
```

Comment:
Good eye, it's likely that it was just missed in previous designs- I would suggest adding the spacing and raising a bug/code maintenance to track the other pages so that it can be updated to the same spec. Perhaps one for the Frontend channel?

Reply:
I added an open point for that: https://community.openproject.org/projects/openproject/work_packages/56057/activity 
I remember people telling me they preferred that to discussions in elements, but to be honest it feels a bit exaggerated afterwards now :sweat_smile: 

However, I assume we can probably keep it as is or the styles will be adapted globally and I can add that afterwards. So I think this topic this should not block this PR.

Reply:
It turned out that people want the more spacious spacing and I created a separate ticket for that: https://community.openproject.org/projects/openproject/work_packages/56060